docs/modules/ROOT/pages/explanations/decisions/postgresql.adoc

```diff
@@ -0,0 +1,84 @@
+= PostgreSQL Operators
+
+== Problem
+
+Creating our own operator to deploy and manage PostgreSQL proved to be a gargantuan effort. In order to keep in line with the newly specified https://kb.vshn.ch/app-catalog/reference/framework-requirements.html#_for_service_maintainer["more buy than make"] approach, we're evaluating the viability of using existing operators for the use of converged services.
+
+== Evaluated Operators
+
+[cols="1,1,1,1,1,1,1"]
+|===
+|Requirements |Percona |Stackgres |CloudnativePG |Zalando |Crunchydata |Bitnami Helmchart
+|Superuser Access |✅ |✅ |✅ |✅ |✅ |✅
+
+|Database Management |❌ |❌ |❌ |✅ |✅ |❌
+
+|User Management |❌ |❌ |❌ |✅ |✅ |✅
+
+|Service Metrics |❌ |✅ |✅ |❌ |✅ |✅
+
+|Maintenance Schedule |✅ |✅ |❌ |❌ |❌ |❌
+
+|Backup Schedule |✅  |✅  |✅  |✅  |✅  |❌
+
+|Self Service Backup Restore |✅ |✅ |✅ |✅ |✅ |❌
+
+|Encryption at rest |✅ |✅ |✅ |✅ |✅ |✅
+
+|Connection limit of 25/Gb Memory |✅ |✅ |✅ |❌ |✅ |✅
+
+|Extension Management |❌ |✅ |❌ |✅ |❌ |✅
+
+|Custom PostgreSQL Settings |✅ |✅ |✅ |❌ |✅ |✅
+
+|In-Place Major Upgrade |✅ |✅ |❌ |❌ |❌ |❌
+
+|License |Apache License 2.0 |AGPL-3.0 |Apache License 2.0 |MIT |Apache License 2.0 | Apache License 2.0
+
+|Upstream Support |Some support available not clear if for operators |✅ |✅ (via EDB) |❌ |✅ |❌
+|===
+
+https://wiki.vshn.net/display/VHT/PostgreSQL+Operator+Evaluations[More detailed] comparison is in our internal wiki.
+
+For the POC on which this decision is based on, we opted to test out Stackgres. It's feature set and general feel were good during the evaluation phase.
+
+== Pros
+
+* Brings cluster functionality
+* Backups out of the box, incl. PIT
+* Auto deploys Prometheus service monitors
+* Brings Dbops CRs that handle some operations in a k8s native way
+* Security update schedules can be implemented
+* The operator is maintained by a third party
+* Commercial support available
+
```

Comment:
```suggestion
* Upgrades to the operator affect all users at the same time, independent of their maintenance window settings.

```

Reply:
At least for stackgres this statement is not true. It won't restart anything until explicitly done via a DbOps CR.

Reply:
I wasn't referring to DB restarts, but in general if there are breaking changes (either in API or behavior). I guess this isn't only applicable to Stackgres, but for all operators.

Reply:
Ah then I suggest to rephrase it:

`Upgrading the operator could inntroduce breaking changes`

To some degree we can handle those changes in the compositions. But a good point we should be aware about.